CHANGELOG.md

```diff
@@ -7,13 +7,14 @@
 * Remove deprecated ``loop`` parameters.
 * Remove deprecated ``cache`` parameter from ``create()``.
 * Use ``base._ensure_key()`` in ``_build_key()`` to ensure consistency of enum
-keys between different Python versions. [#633](https://github.com/aio-libs/aiocache/issues/633) -- Padraic Shafer
```

Comment:
Double hyphen is the correct format:
https://github.com/aio-libs/aiohttp/tree/master/CHANGES#alright-so-how-to-add-a-news-fragment